docs/faqs/max_execution_time.rst

```diff
@@ -8,31 +8,58 @@ To ensure fairness, there is a maximum execution time for each Qiskit Runtime jo
 a job exceeds this time limit, it is forcibly cancelled. This is represented in the job
 status as `Cancelled - Ran too long`. The maximum execution time is the
 smaller of 1) the system limit and 2) the ``max_execution_time`` defined by the program.
+
+System limit
+--------------
+
 The system limit is defined as follows:
 
 Qiskit Runtime on IBM Cloud
----------------------------
+***************************
 
 The system limit on the job execution time is 3 hours for a job running on a simulator
 and 8 hours for a job running on a physical system.
 
 Qiskit Runtime on IBM Quantum
------------------------------
+*****************************
 
-The system limit on the job execution time is
+The system limit on the job execution time is described in the following table:
 
 +------------------+--------------+-----------+--------------+-----------+
 |                  | Public Program           | Private Program          |
 +==================+==============+===========+==============+===========+
 |                  | Premium User | Open User | Premium User | Open User |
 +------------------+--------------+-----------+--------------+-----------+
-| Simulated Device | 3h           | 1h        | 3h           |1h         |
+| Simulated Device | 3h           | 1h        | 3h           | N/A       |
 +------------------+--------------+-----------+--------------+-----------+
-| Real Device      | 8h           | 4h        | 8h           |2h         |
+| Real Device      | 8h           | 4h        | 8h           | N/A       |
 +------------------+--------------+-----------+--------------+-----------+
 
 Note that a *premium user* here means a user who has access to backends in providers other than ibm-q/open/main.
 
+Max execution time
+------------------
+
+There are several factors that impact the program's maximum execution time. In addition to the ``max_execution_time`` parameter, there are time limits on programs and sessions.
```

Comment:
I'm not sure if this is meant to say job's max execution time or program's. I think it should be job's, since line 10 talked about system, program, and [to be added] job's limits. 

Then we can describe how one can set a max execution time for an individual job, and show them code snippet on how to do it.